Bottom: d57e3afd8921c3acb0cf93ffb59125d6e9477c5a
Top:    74fb4ec10e731d25aec8f1398d2579c0a0054016
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-12-18 21:47:25 +0000

Refresh of hid-gtk-fix-copy-paste-error-b

---

diff --git a/src/hid/gtk/gui-netlist-window.c b/src/hid/gtk/gui-netlist-window.c
index 35b5427..f87491d 100644
--- a/src/hid/gtk/gui-netlist-window.c
+++ b/src/hid/gtk/gui-netlist-window.c
@@ -295,7 +295,7 @@ node_selection_changed_cb (GtkTreeSelection * selection, gpointer data)
         {
           PadTypePtr pad = (PadTypePtr) conn.ptr2;
           x = pad->Point1.X + (pad->Point2.X - pad->Point1.X) / 2;
-          y = pad->Point1.Y + (pad->Point2.Y - pad->Point2.Y) / 2;
+          y = pad->Point1.Y + (pad->Point2.Y - pad->Point1.Y) / 2;
           gui->set_crosshair (x, y, 0);
           ghid_lead_user_to_location (x, y);
           break;
